### ISSUE TITLE
Timezone and whazzupsource

### ISSUE THREAD

**Issue by rteunissen** [donatmarko/ivao-tracker-backend#3]
With the current timezone being set, the script gives errors in the log: 

2020-01-15 05:26:42     ERROR   1298 (HY000): Unknown or incorrect time zone: 'Etc/UTC'
NoneType: None

Once i overcome that (by outcommenting the line in main.py) it has some issues with my whazzup feed. In the config file i just put the url to whazzup.txt, but i have the feeling it wants something in json. Here's the logs: 

2020-01-15 05:40:53     INFO    Getting Whazzup feed...
2020-01-15 05:40:53     DEBUG   Starting new HTTP connection (1): api.ivao.aero
2020-01-15 05:40:53     DEBUG   http://api.ivao.aero:80 "GET /getdata/whazzup/whazzup.txt HTTP/1.1" 200 None
2020-01-15 05:40:53     ERROR   Expecting value: line 1 column 1 (char 0)
NoneType: None

Does it need it in json? And if so, is there a direct url for this, or do i just need something that converts the txt to json?

**Comment by donatmarko** [comment]
Hello,
MySQL timezone data shall be loaded in order the Etc/UTC work properly.
https://dba.stackexchange.com/questions/120945/how-do-i-resolve-this-error-error-1298-hy000-unknown-or-incorrect-time-zone

Regarding the data source, yes, it needs it json formatted. You may use the following, and I'm going to publish the converter script in a different repo in the next few days.
https://api.donatus.hu/oavi/json/whazzup.json

**Comment by rteunissen** [comment]
I got it working now it seems, but there are some issues i stumbled upon in the database scheme. 

ERROR   1364 (HY000): Field 'disconnected_at' doesn't have a default value
The disconnected_at fields are now set to NOT_NULL. If you change that so that it can be NULL, that one is fixed. 

ERROR   1406 (22001): Data too long for column 'fp_aircraft' at row 1
The field was set to Varchar(30), which can be kinda tight sometimes. I would suggest to go to at least 100, or maybe even more. I've set it to 300 for myself now. 

Thank you for your help, it all seems to work for me now :)
